handlers/TileMapHandler.js

```diff
@@ -244,6 +246,44 @@ class TileMapHandler {
         }
     }
 
+    resetPlayerStartPoint() {
+        const levelObjects = this.levelObjects;
+        const startFlagsInLevel = levelObjects.filter(levelObject => levelObject.type === ObjectTypes.START_FLAG);
+        const doorsInLevel = levelObjects.filter(levelObject => levelObject.type === ObjectTypes.DOOR);
+        const defaultStartFlag = startFlagsInLevel.find(startFlag => startFlag.extraAttributes?.defaultStartFlag);
+        const fallbackStartFlag = startFlagsInLevel[startFlagsInLevel.length - 1];
```

Comment:
startFlagsInLevel length could be 0, and the index could be -1 then, or am i wrong?

Reply:
You’re correct, yeah—in that case `fallbackStartFlag` will end up being `undefined` because there aren’t any start flags in the level. Currently this is handled further down below by printing a warning to the console when no suitable start location is found. In practice, this occurs when you have a level that is only accessible via door, and will trigger every time you access such a level in build mode. 
This can actually be pretty common/easy to encounter, so maybe this behavior should be revised a bit to accept Doors as a fallback too. (Or to give Doors the same `defaultStartFlag` attribute as StartFlags, but that seems a bit overkill?)